src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header a {
```

Comment:
do not increase selectors specificity for no reason, read [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

Reply:
In this case I should make a selector to tag `a`, am I right?

Reply:
Use a classes selector instead of the tag selector